build.gradle.kts

```diff
@@ -36,6 +36,9 @@ subprojects {
         maven {
             url = uri("https://maven.iais.fraunhofer.de/artifactory/eis-ids-public/")
         }
+        maven {
+            url = uri("https://jitpack.io")
```

Comment:
For which dependency is this used? Do we need jitpack.io at all? If we don't I'd argue to remove this repo since it somehow undermines the whole purpose of having signed maven release artifacts coming from trustworthy sources such as maven central.

Reply:
I added it because of `multihash`, which should be available in mavencentral, but somehow is not found there? will investigate further.